test/test_gtype.ml

```diff
@@ -1,3 +1,4 @@
+(*
```

Comment:
This file is all commented out, do we really need it?

Reply:
https://github.com/nuscr/nuscr/commit/b7ff56477c7b1d58553e702f467ca61b6cff5e6a